app/views/content.js

```diff
@@ -0,0 +1,40 @@
+import Mn from 'backbone.marionette';
+import HeaderView from './header';
+import TracksView from './tracks';
+import application from '../application';
+
+
+const Content = Mn.LayoutView.extend({
+    
+    template: require('./templates/content'),
+
+    regions: {
+        header: '[role="header"]',
+        tracks: '[role="tracks"]',
+    },
+
+    onBeforeShow: function() {
+        this.showChildView('header', new HeaderView());
+        this.switchPlaylist(application.allTracks);
+    },
+
+    switchPlaylist: function (collection) {
+        console.log(collection)
+        this.showChildView('tracks',
+            new TracksView({ collection: collection})
+        );
+        this.stopListening();
+        application.headerInfos.set('count', collection.length);
```

Comment:
I suggest, instead of setting here the `headerInfos` concerns related to the current collection, that the HeaderView component itself listen to the `playlist:switch` event (which payload the collection) and rebind itself yupon the new collection when it catches the event.